File: m/rls/rlb/wcmai.c
/* file -  wcmai - default wc_mai */
#include "m:\rid\rider.h"
#include "m:\rid\wcdef.h"
#include "m:\rid\cldef.h"
#include "m:\rid\imdef.h"
extern int _main (int ,char **);
/* code -  wc_mai - cusp main */
wc_mai(
wsTevt *evt )
{ char *cmd = ws_str (evt, wsCMD);
  int cnt = 0;
  char *(*vec )= NULL;
  cl_vec (cmd, &cnt, &vec);
  _main (cnt, vec);
  im_exi ();
} 

File: m/rls/wcmai.r
file	wcmai - default wc_mai
include	rid:rider
include	rid:wcdef
include	rid:cldef
include	rid:imdef

;	These routines are inserted into the rider window library.
;	They are only called if the corresponding user routine is missing.
;	The library must be built with non-exclusive module names.
;	Each must occupy a separate module

code	wc_mai - cusp main

  func	wc_mai
	evt : * wsTevt
  is	fine
  end
